pkg/endpoint/endpoint.go

```diff
@@ -2128,18 +2128,18 @@ func (e *Endpoint) Delete(monitor monitorOwner, ipam ipReleaser, manager endpoin
 		}
 	}
 
-	if option.Config.IPAM == option.IPAMENI {
+	if option.Config.IPAM == option.IPAMENI || option.Config.IPAM == option.IPAMAzure {
 		e.getLogger().WithFields(logrus.Fields{
 			"ep":     e.GetID(),
 			"ipAddr": e.GetIPv4Address(),
-		}).Debug("Deleting endpoint ENI rules")
+		}).Debug("Deleting endpoint interface rules")
```

Comment:
```suggestion
		}).Debug("Deleting endpoint routing rules")
```
I like "routing" better here as "interface rules" isn't very informative